backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -308,3 +313,43 @@ def verbose_instance_factory(self):
 
 #     def verbose_instance_factory(self):
 #         return factories.ContentItemOrderFactory()
+
+
+class WorkshopAttendanceViewsetTests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "workshopattendance-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+
+    def verbose_instance_factory(self):
+        workshop_attendance = factories.WorkshopAttendanceFactory()
+        content = workshop_attendance.content_item
+        content.topic_needs_review = False
+        content.save()
+        return workshop_attendance
+
+    def test_get_list(self):
```

Comment:
`def test_get_list(self):` is a bunction that already exists on the base class. If you write another function of the same name then you override the behavior in the base class. Is there a good reason to override the existing test functionality with tis new function?

It looks like your `test_get_list` function tests a bunch of stuff that has nothing to do with getting any lists. And it also doesn't do a bunch of the checks set out in the base class.